### ISSUE TITLE
pom.xml links to missing version of jackson-bom

### ISSUE THREAD

**Issue by rossspencer715** [redouane59/twittered#427]
Hi,

Not sure if I'm just missing something since I'm not terribly familiar with Maven yet and am trying to use this library in a school project, so apologies if this is a me problem. 

If I try to sync my build.gradle app file with a dependency for Twittered versions 2.20 and 2.21, I run into:
```
Required by:
         project :app > io.github.redouane59.twitter:twittered:2.21 > com.fasterxml.jackson.core:jackson-databind:2.13.4.1
   > Could not find com.fasterxml.jackson:jackson-bom:2.13.4.1.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
```

The closest version I see on maven-central is 2.13.4.20221013, with no entry matching 2.13.4.1. 
https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/

Reverting to v2.19 works perfectly since it's able to resolve the older version (looks like it changed here: https://github.com/redouane59/twittered/commit/5832190859073d023bd5809e478d2be55c4aa16b). 2.13.4.1 is listed on the jackson website but appears to be missing from maven, not sure if it got pulled or what.

Looking forward to using the new DM stuff eventually! This library is fantastic so far. Thank y'all for the hard work -- please let me know if this is just something I'm doing wrong or if a PR to modify to v2.13.4 would be helpful in the meantime

**Comment by redouane59** [comment]
Hey @rossspencer715 , it looks like it comes from here :
https://github.com/redouane59/twittered/commit/5832190859073d023bd5809e478d2be55c4aa16b

On my side the build is working. In maven, the version 2.13.4 of the library exists here : 
https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-core/2.13.4
I don't know if the .1 after can be added by default without impact if using Maven. 

Can you try to change `2.13.4.1` by `2.13.4` ?

**Comment by steve1316** [comment]
I also had the same problem with `implementation("io.github.redouane59.twitter:twittered:2.21")` and `implementation("io.github.redouane59.twitter:twittered:2.20")` giving the same error. Downgrading down to `2.19` gets rid of the error and will let you build.

@redouane59, what did you mean by:
> Can you try to change 2.13.4.1 by 2.13.4 and tell me if it works ?

Did you mean do this in the app's build.gradle:
```
implementation("io.github.redouane59.twitter:twittered:2.21")
implementation("com.fasterxml.jackson:jackson-bom:2.13.4")
```

Same error as before: 
```
Could not find com.fasterxml.jackson:jackson-bom:2.13.4.1.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - file:/C:/Users/steve1316/.m2/repository/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - https://jcenter.bintray.com/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - file:/C:/Users/steve1316/Documents/GitHub/granblue-automation-android/node_modules/react-native/android/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - file:/C:/Users/steve1316/Documents/GitHub/granblue-automation-android/node_modules/jsc-android/dist/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - https://dl.google.com/dl/android/maven2/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - https://www.jitpack.io/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
```

**Comment by redouane59** [comment]
Yes I meant that. Really strange, can't understand why this is not working while the upgrade version was pushed automatically.
